Examples/ObjectiveC/LightExample.m

```diff
@@ -0,0 +1,70 @@
+//
+//  LightExample.m
+//  Examples
+//
+//  Created by Jordan Kiley on 6/22/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+
+#import "LightExample.h"
+@import Mapbox;
+
+@interface LightExample () <MGLMapViewDelegate>
+
+@end
+
+NSString *const MBXExampleLight = @"LightExample";
+
+@implementation LightExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Set the map style to Mapbox Light Style version 9. The map's source will be queried later in this example.
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURLWithVersion:9]];
+    mapView.delegate = self;
+    
+    // Center the map on the Flatiron Building in New York, NY.
+    mapView.camera = [MGLMapCamera cameraLookingAtCenterCoordinate:CLLocationCoordinate2DMake(40.7411, -73.9897) fromDistance:600 pitch:45 heading:200];
+    mapView.tintColor = [UIColor grayColor];
+    
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    
+    // Add a MGLFillExtrusionStyleLayer.
+    [self addFillExtrusionLayer:style];
+    
+    // Create an MGLLight object.
+    MGLLight *light = [[MGLLight alloc] init];
+    
+    // Create an MGLSphericalPosition and set the radial, azimuthal, and polar values.
```

Comment:
For those who aren’t very mathematically inclined I’m not sure this sentence has much meaning. 😉

I see that the docs for [`MGLSphericalPosition`](https://www.mapbox.com/ios-sdk/api/3.6.0-rc.1/Structs/MGLSphericalPosition.html) have short explanations of these params, but perhaps we could also bring even more concise (one sentence each?) versions here.